crates/core/src/commands/check.rs

```diff
@@ -37,7 +37,54 @@ pub struct CheckOptions {
     pub read_data: bool,
 }
 
-impl CheckOptions {
+#[derive(Debug, PartialEq, Eq)]
+#[non_exhaustive]
+/// `CheckErrorLevel` describes severity levels of problems identified by check.
+pub enum CheckErrorLevel {
```

Comment:
I would probably stick to the `kind` naming scheme we have in rustic_core, so something like `CheckErrorKind`.

Reply:
Not relevant, misread it